src/main/java/org/fundacionjala/coding/richard/MostFrequent.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.coding.richard;
+
+/**
+ * Created by Administrator on 9/5/2017.
+ */
+public class MostFrequent {
+
+    /**
+     * This method returns the most frequent item count.
+     *
+     * @param array This variable contains the array of ints.
+     * @return return the most frequent item count.
+     */
+    public int mostFrequentItemCount(int[] array) {
+        if (array.length == 0) {
+            return 0;
+        }
+        int count = 0;
+        int biggerNumber = 0;
+        for (int anArray1 : array) {
+            for (int anArray : array) {
```

Comment:
please use correct names
I don't understand your code

Reply:
Ok, I am going to solve that.